phoenicis-javafx/src/main/java/org/phoenicis/javafx/controller/containers/ContainersController.java

```diff
@@ -34,6 +34,9 @@
 import org.phoenicis.repository.dto.ApplicationDTO;
 import org.phoenicis.repository.dto.RepositoryDTO;
 
+import java.awt.*;
```

Comment:
Do we really require `awt` for this?
Is there no way to do this with javafx?

Reply:
I searched for it. If there is a way, I couldn't find it.

Reply:
This is to open something in the default browser. I'm not sure it's a good idea. Check also the comment in the second link:

> On my computer starts the default browser and shows the PDf file in the browser. Why? (Host is Linux Mint).

Reply:
Yes, you're right.
I'm fine with merging this for now.
But this solution is not really platform independent, because it runs only on standard JVMs which provide AWT, which Android for example doesn't.

As a long term solution, maybe we can implement a similar solution to our terminal opener?